app/views/shared/_header.html.erb

```diff
@@ -9,47 +9,43 @@
   navbar-default
   navbar-inverse
   navbar-static-top
-  <% if !t('admin.whats_new.show_banner') && admin_template %>add-bottom-margin<% end %>
-  <% if environment_style %>environment-indicator<% end %>" role="banner">
```

Comment:
I think we should remove the `<header>` element at this stage so that we do not nest one of these elements within another, which could affect the way it renders.

I don't think it matters that this will also remove the navigation from header because we are also going to have to adopt a slightly different structure to the existing one when we use the `layout_header` component, so that the navigation is not structurally part of the header but is a separate `<nav>` element (but that is part of the next piece of work).

Reply:
@davidtrussler 
As discussed I have removed the header tag and update the design view. Now the view will look like below and it will get fixed once sub-navigation will get implemented.

![image](https://github.com/alphagov/whitehall/assets/131259138/b14e039a-24a4-4d81-a202-320a3b3cf36b)